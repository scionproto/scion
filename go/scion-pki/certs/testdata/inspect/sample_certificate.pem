-----BEGIN CERTIFICATE-----
MIIC6zCCApCgAwIBAgIUKYAlHNyKuRUola3adT9IZRc/V3IwCgYIKoZIzj0EAwQw
gbkxCzAJBgNVBAYTAkNIMRAwDgYDVQQIDAdaw7xyaWNoMRAwDgYDVQQHDAdaw7xy
aWNoMRUwEwYDVQQKDAwxLWZmMDA6MDoxMTAxIzAhBgNVBAsMGjEtZmYwMDowOjEx
MCBJbmZvU2VjIFNxdWFkMSswKQYDVQQDDCIxLWZmMDA6MDoxMTAgU2VjdXJlIENB
IENlcnRpZmljYXRlMR0wGwYLKwYBBAGDsBwBAgEMDDEtZmYwMDowOjExMDAeFw0y
MTAzMTgxNzEyMzFaFw0yMjAzMTgxNzEyMzFaMIGyMQswCQYDVQQGEwJDSDEQMA4G
A1UECAwHWsO8cmljaDEQMA4GA1UEBwwHWsO8cmljaDEVMBMGA1UECgwMMS1mZjAw
OjA6MTEwMSMwIQYDVQQLDBoxLWZmMDA6MDoxMTAgSW5mb1NlYyBTcXVhZDEkMCIG
A1UEAwwbMS1mZjAwOjA6MTEwIEFTIENlcnRpZmljYXRlMR0wGwYLKwYBBAGDsBwB
AgEMDDEtZmYwMDowOjExMDBZMBMGByqGSM49AgEGCCqGSM49AwEHA0IABAwP3DZ3
vWNrH6VP9xh5dyixYlt4/+aKcwEBGehDlV4D+AhVhPEky6mGc+KVmEobhbPhwCBW
Zj++2Cqt4yLGtpWjezB5MA4GA1UdDwEB/wQEAwIHgDAdBgNVHQ4EFgQUKr3/r9NY
5NI5ONz1j7WRQLkyL6kwHwYDVR0jBBgwFoAUYLb1HFxm2+P7FzE8bsOR45LoLfEw
JwYDVR0lBCAwHgYIKwYBBQUHAwEGCCsGAQUFBwMCBggrBgEFBQcDCDAKBggqhkjO
PQQDBANJADBGAiEAx5NARijIoegY+L8HuU5vOxTc/8/EnktjDGhRCVswJ0MCIQCc
tgFZLg0aBaSEoxzH5Zo8VIQDC8koA5G+LrtOaH+4DQ==
-----END CERTIFICATE-----
-----BEGIN CERTIFICATE-----
MIIC5jCCAougAwIBAgIUdNfmfI4qApOyfSt4sOcA7tx3ITYwCgYIKoZIzj0EAwQw
gcIxCzAJBgNVBAYTAkNIMRAwDgYDVQQIDAdaw7xyaWNoMRAwDgYDVQQHDAdaw7xy
aWNoMRUwEwYDVQQKDAwxLWZmMDA6MDoxMTAxIzAhBgNVBAsMGjEtZmYwMDowOjEx
MCBJbmZvU2VjIFNxdWFkMTQwMgYDVQQDDCsxLWZmMDA6MDoxMTAgSGlnaCBTZWN1
cml0eSBSb290IENlcnRpZmljYXRlMR0wGwYLKwYBBAGDsBwBAgEMDDEtZmYwMDow
OjExMDAeFw0yMTAzMTgxNzEyMzFaFw0yMzAzMTgxNzEyMzFaMIG5MQswCQYDVQQG
EwJDSDEQMA4GA1UECAwHWsO8cmljaDEQMA4GA1UEBwwHWsO8cmljaDEVMBMGA1UE
CgwMMS1mZjAwOjA6MTEwMSMwIQYDVQQLDBoxLWZmMDA6MDoxMTAgSW5mb1NlYyBT
cXVhZDErMCkGA1UEAwwiMS1mZjAwOjA6MTEwIFNlY3VyZSBDQSBDZXJ0aWZpY2F0
ZTEdMBsGCysGAQQBg7AcAQIBDAwxLWZmMDA6MDoxMTAwWTATBgcqhkjOPQIBBggq
hkjOPQMBBwNCAATp4pL81fib73MCFmQ+6NgVbzFuOibeLwLVsdcs/rc6lwZ78dQ9
qRSTDgTVyFECzdUPB6OLoWD+11XmNRWDPi0So2YwZDASBgNVHRMBAf8ECDAGAQH/
AgEAMA4GA1UdDwEB/wQEAwICBDAdBgNVHQ4EFgQUYLb1HFxm2+P7FzE8bsOR45Lo
LfEwHwYDVR0jBBgwFoAUl8DWMCfTA6W7IYnTUDtFoTwg03IwCgYIKoZIzj0EAwQD
SQAwRgIhAMQIXHPhhwNTc9WRkoWRALcPXfDCCE8ZFaZ3fqhNlM0oAiEAg4VN+HN7
P2JS3EVZGDFKTZp9vkGTW8giguR5+gG38u0=
-----END CERTIFICATE-----
