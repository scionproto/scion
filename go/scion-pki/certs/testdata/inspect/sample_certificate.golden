Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number: 236926349964825539132366110936757753936879769458 (0x2980251cdc8ab9152895adda753f4865173f5772)
    Signature Algorithm: ECDSA-SHA512
        Issuer: C=CH,ST=Zürich,L=Zürich,O=1-ff00:0:110,OU=1-ff00:0:110 InfoSec Squad,CN=1-ff00:0:110 Secure CA Certificate,UnknownOID=1.3.6.1.4.1.55324.1.2.1
        Validity
            Not Before: Mar 18 17:12:31 2021 UTC
            Not After : Mar 18 17:12:31 2022 UTC
        Subject: C=CH,ST=Zürich,L=Zürich,O=1-ff00:0:110,OU=1-ff00:0:110 InfoSec Squad,CN=1-ff00:0:110 AS Certificate,UnknownOID=1.3.6.1.4.1.55324.1.2.1
        Subject Public Key Info:
            Public Key Algorithm: ECDSA
                Public-Key: (256 bit)
                X:
                    0c:0f:dc:36:77:bd:63:6b:1f:a5:4f:f7:18:79:77:
                    28:b1:62:5b:78:ff:e6:8a:73:01:01:19:e8:43:95:
                    5e:03
                Y:
                    f8:08:55:84:f1:24:cb:a9:86:73:e2:95:98:4a:1b:
                    85:b3:e1:c0:20:56:66:3f:be:d8:2a:ad:e3:22:c6:
                    b6:95
                Curve: P-256
        X509v3 extensions:
            X509v3 Key Usage: critical
                Digital Signature
            X509v3 Subject Key Identifier:
                2A:BD:FF:AF:D3:58:E4:D2:39:38:DC:F5:8F:B5:91:40:B9:32:2F:A9
            X509v3 Authority Key Identifier:
                keyid:60:B6:F5:1C:5C:66:DB:E3:FB:17:31:3C:6E:C3:91:E3:92:E8:2D:F1
            X509v3 Extended Key Usage:
                Server Authentication, Client Authentication, Time Stamping
    Signature Algorithm: ECDSA-SHA512
         30:46:02:21:00:c7:93:40:46:28:c8:a1:e8:18:f8:bf:07:b9:
         4e:6f:3b:14:dc:ff:cf:c4:9e:4b:63:0c:68:51:09:5b:30:27:
         43:02:21:00:9c:b6:01:59:2e:0d:1a:05:a4:84:a3:1c:c7:e5:
         9a:3c:54:84:03:0b:c9:28:03:91:be:2e:bb:4e:68:7f:b8:0d
Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number: 667057667842995775975688328585225864023964590390 (0x74d7e67c8e2a0293b27d2b78b0e700eedc772136)
    Signature Algorithm: ECDSA-SHA512
        Issuer: C=CH,ST=Zürich,L=Zürich,O=1-ff00:0:110,OU=1-ff00:0:110 InfoSec Squad,CN=1-ff00:0:110 High Security Root Certificate,UnknownOID=1.3.6.1.4.1.55324.1.2.1
        Validity
            Not Before: Mar 18 17:12:31 2021 UTC
            Not After : Mar 18 17:12:31 2023 UTC
        Subject: C=CH,ST=Zürich,L=Zürich,O=1-ff00:0:110,OU=1-ff00:0:110 InfoSec Squad,CN=1-ff00:0:110 Secure CA Certificate,UnknownOID=1.3.6.1.4.1.55324.1.2.1
        Subject Public Key Info:
            Public Key Algorithm: ECDSA
                Public-Key: (256 bit)
                X:
                    e9:e2:92:fc:d5:f8:9b:ef:73:02:16:64:3e:e8:d8:
                    15:6f:31:6e:3a:26:de:2f:02:d5:b1:d7:2c:fe:b7:
                    3a:97
                Y:
                    06:7b:f1:d4:3d:a9:14:93:0e:04:d5:c8:51:02:cd:
                    d5:0f:07:a3:8b:a1:60:fe:d7:55:e6:35:15:83:3e:
                    2d:12
                Curve: P-256
        X509v3 extensions:
            X509v3 Basic Constraints: critical
                CA:TRUE, pathlen:0
            X509v3 Key Usage: critical
                Certificate Sign
            X509v3 Subject Key Identifier:
                60:B6:F5:1C:5C:66:DB:E3:FB:17:31:3C:6E:C3:91:E3:92:E8:2D:F1
            X509v3 Authority Key Identifier:
                keyid:97:C0:D6:30:27:D3:03:A5:BB:21:89:D3:50:3B:45:A1:3C:20:D3:72
    Signature Algorithm: ECDSA-SHA512
         30:46:02:21:00:c4:08:5c:73:e1:87:03:53:73:d5:91:92:85:
         91:00:b7:0f:5d:f0:c2:08:4f:19:15:a6:77:7e:a8:4d:94:cd:
         28:02:21:00:83:85:4d:f8:73:7b:3f:62:52:dc:45:59:18:31:
         4a:4d:9a:7d:be:41:93:5b:c8:22:82:e4:79:fa:01:b7:f2:ed
